web/src/styles.css

```
@@ -19,6 +21,11 @@
 
   --border: var(--token-colors-border-default);
   --shadow-strong: 0 24px 50px var(--token-colors-alpha-shadow);
+  color: var(--text-main);
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)